### PR TITLE
add login functionality

### PR DESCRIPTION
**Pull request by megschuetz**
This PR will take in the login info from a user and use their ID from the username to find that user in the traveler data. Then their ID is set to a global variable to be used in several places when loading the main page. Look to the scripts file (mostly at the top). Also made some small CSS changes.

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -7,184 +7,225 @@ import {travelersData, destinationsData, tripsData, addNewTrip, fetchData, trips
...
-  datePicker.min = dayjs(today).add(1, 'day').format('YYYY-MM-DD')
-}
+  today = dayjs(Date()).format('YYYY-MM-DD');
+  datePicker.min = dayjs(today).add(1, 'day').format('YYYY-MM-DD');
+};
 
 function buildDropDown() {
-  let numbers = ''
+  let numbers = '';
   for (let i = 1; i < 51; i++) {
```
Comment (kendallm360) [inline src/scripts.js:172]: I really like the dropdowns on your site, again great example of making the app as easy to use as possible for the user

File: dist/index.html [hunk 2]
```diff
@@ -57,12 +53,15 @@ <h4 class="hidden">Click Below to Confirm</h4>
...
     </section>
-    <footer>
-      <p id="total-cost-this-year"></p>
-      <p class="spent-this-year">Spent This Year</p>
+    <footer class="hidden">
+      <div class="total">
+        <p id="total-cost-this-year"></p>
+        <p class="spent-this-year">Spent This Year</p>
+      </div>
+      <p class="fee">*includes a 10% travel agent fee</p>
```
Comment (kendallm360) [inline dist/index.html:64]: I probably should also tell my user that this is included in their estimated price.

File: src/css/styles.css [hunk 3]
```diff
@@ -36,54 +34,29 @@ h3 {
...
-  right: 10px; 
-  width: 15px;
-  height: 15px;
-  background: #888;
-  color: #EEE;
-  text-align: center;
-  line-height: 15px;
-  border-radius: 15px;
-  cursor: pointer;
+  to {
```
Comment (kendallm360) [inline src/css/styles.css:57]: Great job making your site responsive in multiple areas

File: src/css/styles.css [hunk 4]
```diff
@@ -262,7 +232,29 @@ footer {
...
+
+.fee {
+  font-size: 11px;
+  margin-right: 17px;
+  margin-top: 5px;
+  height: 25px;
+}
+
+.hidden {
+  display: none !important;
```
Comment (kendallm360) [inline src/css/styles.css:259]: I am curious if this works without `!important`, if not I would like to look into why it is necessary for my own learning

**Comment by kendallm360** [comment]
I am super impressed, your code and site looks great. I have not worked with you in a group or breakout room but seeing this gives me the impression you have a good handle on things. Keep up the good work

**Review by kendallm360 — APPROVED** [review]

File: src/css/styles.css (hunk 4 above)
Comment (megschuetz) [reply src/css/styles.css:259]: I just pulled this from a previous project, and actually don't know, so that is a good a question and I will look into it!

**Pull request merged**